proto/substrait/algebra.proto

```diff
@@ -410,6 +412,13 @@ message DdlRel {
   // The body of the CREATE VIEW
   Rel view_definition = 7;
 
+  // Field that could be used to extend or introduce any target specific metadata about the table,
+  // like replication, partition, etc.
+  ExtensionObject table_metadata = 8;
```

Comment:
There's an ExtensionObject above which deals with what we're operating on.  Is that different than what would be used here?  Do you see this useful in conjunction with a named_object?

Reply:
It looks like that the ExtensionObject inside oneof write_type field
intended for a somewhat different purpose
not for general information about table definitions / metadata
but indeed this information can also be transmitted via ExtensionObject in oneof write_type